pokemongo_bot/walkers/polyline_generator.py

```diff
@@ -49,6 +51,7 @@ def cached_polyline(origin, destination, speed, google_map_api_key=None):
                 PolylineObjectHandler._run = True
                 PolylineObjectHandler._instability = 20  # next N moves use same cache
 
+            speed = min(speed or PolylineObjectHandler._standard_average_speed, PolylineObjectHandler._standard_max_speed)
```

Comment:
If someone wonders why the hardcoded values... it does not matter that much... the SpetWalker.step(speed=x) will override those values... 

this values are used only for calculating the nr. of elevation samples... not for walking!